src/components/Button/Button.tsx

```diff
@@ -0,0 +1,9 @@
+import stl from "./ButtonStyle.module.scss";
+
+type ButtonProps = { btnText: string };
+
+const Button = ({ btnText }: ButtonProps) => {
+  return <div className={stl.button}>{btnText}</div>;
```

Comment:
Use <button/> element instead of <div/>.

Reply:
Implemented